.changeset/wise-poems-do.md

```diff
@@ -0,0 +1,5 @@
+---
+"openapi-fetch": minor
```

Comment:
```suggestion
"openapi-fetch": patch
```

For pre-1.0 versions, minor versions should indicate breaking changes. A patch would be better here, as requested